openedx/core/djangoapps/geoinfo/middleware.py

```diff
@@ -33,14 +33,18 @@ def process_request(self, request):
         new_ip_address = get_real_ip(request)
         old_ip_address = request.session.get('ip_address', None)
 
+        reader = geoip2.database.Reader(settings.GEOIP_PATH)
```

Comment:
I think this can be moved down into the elif clause below, no?   It looks like it is only used there.